### PR TITLE
Suppressing autorest error for storsimple.json (swagger spec file).

### PR DESCRIPTION
**Pull request by manaas-microsoft**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/storsimple8000series/resource-manager/readme.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ These settings apply only when `--tag=package-2017-06` is specified on the comma
 ``` yaml $(tag) == 'package-2017-06'
 input-file:
 - Microsoft.StorSimple/stable/2017-06-01/storsimple.json
+directive:
```
Comment (dsgouda) [inline specification/storsimple8000series/resource-manager/readme.md]: Has this been approved by @salameer or someone from the review committee?

Comment (mozehgir) [reply specification/storsimple8000series/resource-manager/readme.md]: Hey Deepak
I looped you in to the email thread where suppression was granted. :)

**Comment by dsgouda** [comment]
@azuresdkci Test this please

**Comment by dsgouda** [comment]
@lmazuel Can you take a look why the ruby CI builds are failing here.

**Comment by lmazuel** [comment]
@dsgouda Docker API was broken :/ I restarted the job.

**Comment by dsgouda** [comment]
@lmazuel  Still failing... not sure why though 😕

**Comment by lmazuel** [comment]
@dsgouda The error is related to the PR now:
```shell
INFO:swaggertosdk.autorest_tools:ERROR: Syntax Error Encountered: Syntax error: bad indentation of a mapping entry
INFO:swaggertosdk.autorest_tools:    - file:///git-restapi/specification/storsimple8000series/resource-manager/readme.md:43:4
```

**Comment by dsgouda** [comment]
@azuresdkci Test this please

File: specification/storsimple8000series/resource-manager/readme.md [hunk 2]
```diff
@@ -37,6 +37,10 @@ These settings apply only when `--tag=package-2017-06` is specified on the comma
 ``` yaml $(tag) == 'package-2017-06'
 input-file:
 - Microsoft.StorSimple/stable/2017-06-01/storsimple.json
+directive:
+  -suppress:
```
Comment (dsgouda) [inline specification/storsimple8000series/resource-manager/readme.md]: Please put spaces after `-` here, becomes an invalid yaml otherwise

Comment (manaas-microsoft) [reply specification/storsimple8000series/resource-manager/readme.md]: Added.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**